src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -867,3 +875,40 @@ export default class ClinicalData extends React.Component<
         return this.tabUI.component;
     }
 }
+
+export function toBoxPlotTable(data: any[], labels: string[]) {
```

Comment:
you might as well make this a proper react function component.   search code base for ```React.FunctionComponent``` for example.

Reply:
Fixed in [e15fa21](https://github.com/cBioPortal/cbioportal-frontend/pull/4248/commits/e15fa210a0b187b67329114e1362ecf34f185a76)

Reply:
Moved component to its own file in [4e315ad](https://github.com/cBioPortal/cbioportal-frontend/pull/4248/commits/4e315adaef9966be839cc80e8a742583775532eb)